### ISSUE TITLE
[Bug] XMR / Monero mining form does not show any json data

### ISSUE THREAD

**Issue by QuartzRat** [UnamSanctam/SilentCryptoMiner#2058]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/153561734/89f79a1c-8533-4ef5-a5de-e5bf1abe5b31)
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/153561734/2ac0aca7-ae20-487e-b56f-0243bc58e0ed)

**Comment by UnamSanctam** [comment]
Hmm, that's weird, maybe it somehow stopped working in one of the hotfixes. Thank you for the report.

**Comment by UnamSanctam** [comment]
Of course, the basics is to add your translated strings in this file https://github.com/UnamSanctam/SilentCryptoMiner/blob/master/SilentCryptoMiner/Resources/LocalizedControls.xml.

**Comment by QuartzRat** [comment]
Another question, how do I setup a mining proxy? Is that a easy thing to do?

**Comment by UnamSanctam** [comment]
It's easy, as long as you have somewhere to host one of course: https://github.com/xmrig/xmrig-proxy. Though raising open file limits can be difficult if needed.
